vault/quotas/quotas.go

```diff
@@ -165,7 +165,15 @@ type Manager struct {
 
 	logger     log.Logger
 	metricSink *metricsutil.ClusterMetricSink
-	lock       *sync.RWMutex
+
+	// quotaLock is a lock for manipulating quotas
+	quotaLock *sync.RWMutex
```

Comment:
It's broader than that, maybe phrase it more as this is the lock used when there's not a more specific lock.